nautobot/ipam/migrations/0015_prefix_type.py

```diff
@@ -0,0 +1,68 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import migrations, models
+
+from nautobot.ipam import choices
+
+
+def set_prefix_type(apps, schema_editor):
+
+    Prefix = apps.get_model("ipam", "Prefix")
+    Status = apps.get_model("extras", "Status")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    prefix_ct = ContentType.objects.get_for_model(Prefix)
+
+    try:
+        prefix_default_status = Status.objects.get(content_types=prefix_ct, slug="active")
+    except ObjectDoesNotExist:
+        prefix_default_status = Status.objects.filter(content_types=prefix_ct).exclude(slug="container").first()
+
+    # Set Prefix.type to container for prefixes with status of container
+    Prefix.objects.filter(status__slug="container").update(
+        status=prefix_default_status,
+        type=choices.PrefixTypeChoices.TYPE_CONTAINER,
+    )
+
+    # Set Prefix.type to pool for prefixes with `is_pool=True`
+    Prefix.objects.filter(is_pool=True).update(type=choices.PrefixTypeChoices.TYPE_POOL)
```

Comment:
Might be a good idea to include a couple of `print()` calls here to tell the user what this migration is doing?